### DISCUSSION TITLE
Initiate Slider on pages on which Breeze is excluded

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#149] [1. Help]
Hi,
I am able to initiate slick slider using https://breezefront.com/docs/cms-elements on breeze enabled pages.
But How to initiate slider on the pages which are excluded from breeze.
I am using default magento way like below, slider is generating but whole styling is a mess. Is there any other way to initialize the slider which will use correct styling?

`require(['jquery', 'slick'], function($) {
        $(document).ready(function() {
            $(".product-slider").slick({
                dots: true,
                infinite: true,
                slidesToShow: 3,
                slidesToScroll: 3
            });
        });
    });`

**Comment by vovayatsyuk** [comment]
Are you using BreezeTheme? I think you need to add slick slider styles in this case because BreezeTheme doesn't use slick anywhere.
